tracker-agent/src/main/configuration/cumulocity.json

```diff
@@ -35,7 +36,7 @@
       "port": 80
 
     },
-    "initialDelaySeconds": 220,
+    "initialDelaySeconds": 10,
```

Comment:
I just didn't want to wait 220 seconds for the microservice to start-up.

It doesn't matter because it works with both configuration, 220 is probably overly defensive, the number is arbitrary and cumulocity.json is not used.

Reply:
Ok, I did it. Now every graft have same Kubernetes settings as develop.